File: manager/src/main/java/moe/shizuku/manager/home/LearnMoreViewHolder.java
package moe.shizuku.manager.home;

import android.view.View;

import moe.shizuku.manager.Helps;
import moe.shizuku.manager.R;
import moe.shizuku.manager.utils.CustomTabsHelper;
import rikka.recyclerview.BaseViewHolder;

public class LearnMoreViewHolder extends BaseViewHolder<Object> {

    public static final Creator<Object> CREATOR = (inflater, parent) -> new LearnMoreViewHolder(inflater.inflate(R.layout.item_home_learn_more, parent, false));

    public LearnMoreViewHolder(View itemView) {
        super(itemView);

        itemView.setOnClickListener(v -> CustomTabsHelper.launchUrlOrCopy(v.getContext(), Helps.HOME.get()));
    }
}

File: manager/src/main/java/moe/shizuku/manager/home/LearnMoreViewHolder.kt
package moe.shizuku.manager.home

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import moe.shizuku.manager.Helps
import moe.shizuku.manager.databinding.HomeItemContainerBinding
import moe.shizuku.manager.databinding.HomeLearnMoreBinding
import moe.shizuku.manager.utils.CustomTabsHelper
import rikka.recyclerview.BaseViewHolder
import rikka.recyclerview.BaseViewHolder.Creator

class LearnMoreViewHolder(binding: HomeLearnMoreBinding, root: View) : BaseViewHolder<Any?>(root) {

    companion object {
        val CREATOR = Creator<Any> { inflater: LayoutInflater, parent: ViewGroup? ->
            val outer = HomeItemContainerBinding.inflate(inflater, parent, false)
            val inner = HomeLearnMoreBinding.inflate(inflater, outer.root, true)
            LearnMoreViewHolder(inner, outer.root)
        }
    }

    init {
        root.setOnClickListener { v: View -> CustomTabsHelper.launchUrlOrCopy(v.context, Helps.HOME.get()) }
    }
}
